examples/mwaa/environment.yaml

```
@@ -0,0 +1,272 @@
+apiVersion: mwaa.aws.upbound.io/v1beta1
+kind: Environment
+metadata:
+  annotations:
+    meta.upbound.io/example-id: mwaa/v1beta1/environment
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
```

Please rename the `metadata.names` here and below for this resource, like:

```suggestion
  name: example-environment
```